DISPLAY "(null)" invalid; disabling X11 forwarding
INFO:tensorflow:PS hosts are: ['raiders1:2200']
INFO:tensorflow:Worker hosts are: ['raiders1:2201', 'raiders1:2202', 'raiders1:2203', 'raiders1:2204', 'raiders2:2205', 'raiders2:2206', 'raiders2:2207', 'raiders2:2208', 'raiders3:2209', 'raiders3:2210', 'raiders3:2211', 'raiders3:2212', 'raiders8:2213', 'raiders8:2214', 'raiders8:2215', 'raiders8:2216']
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job ps -> {raiders1:2200}
I tensorflow/core/distributed_runtime/rpc/grpc_channel.cc:206] Initialize HostPortsGrpcChannelCache for job worker -> {raiders1:2201, raiders1:2202, raiders1:2203, raiders1:2204, raiders2:2205, raiders2:2206, raiders2:2207, raiders2:2208, raiders3:2209, raiders3:2210, raiders3:2211, raiders3:2212, raiders8:2213, raiders8:2214, localhost:2215, raiders8:2216}
I tensorflow/core/distributed_runtime/rpc/grpc_server_lib.cc:202] Started server with target: grpc://localhost:2215
INFO:tensorflow:Learning rate: 0.001000, momentum: 0.300000
INFO:tensorflow:Sync mode!!!!!!
INFO:tensorflow:Compute groups : 4
INFO:tensorflow:SyncReplicas enabled: replicas_to_aggregate=16; total_num_replicas=16
INFO:tensorflow:Num nodes per compute groups: 4
INFO:tensorflow:Tokens per step: 4
INFO:tensorflow:2016-07-27 01:55:48.517229 Supervisor
E0727 01:56:00.329874790   47779 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2205': timeout occurred
E0727 01:56:00.329952183   47779 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2206': timeout occurred
E0727 01:56:00.329971932   47779 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2207': timeout occurred
E0727 01:56:00.329993366   47779 tcp_client_posix.c:191]     failed to connect to 'ipv4:172.24.75.41:2208': timeout occurred
E0727 01:56:00.830186307   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 01:56:00.830259236   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 01:56:00.830281261   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 01:56:00.830298003   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 01:56:03.830085922   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 01:56:03.830136532   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 01:56:03.830160161   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 01:56:06.830005876   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 01:56:09.830032564   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 01:56:09.830086809   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 01:56:09.830122489   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 01:56:09.830142600   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 01:56:15.829986858   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 01:56:15.830034761   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 01:56:18.830008546   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 01:56:18.830056900   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 01:56:24.829975445   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 01:56:27.829978442   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 01:56:27.830022378   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 01:56:30.829963220   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 01:56:36.829990301   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 01:56:45.829973891   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 01:56:48.829948319   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
E0727 01:56:51.830082879   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2207': socket error: 113
E0727 01:56:59.053987725   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2206': socket error: 113
E0727 01:57:11.213978183   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2208': socket error: 113
E0727 01:57:24.233933265   47455 tcp_client_posix.c:178]     failed to connect to 'ipv4:172.24.75.41:2205': socket error: 113
run-expr-32node.sh: line 23: 46836 Terminated              CUDA_VISIBLE_DEVICES='' bazel-bin/inception/imagenet_distributed_train --batch_size=16 --data_dir=/lfs/local/0/daniter/16node-data-$2 --job_name='worker' --task_id=$2 --ps_hosts='raiders1:2200' --worker_hosts='raiders1:2201,raiders1:2202,raiders1:2203,raiders1:2204,raiders2:2205,raiders2:2206,raiders2:2207,raiders2:2208,raiders3:2209,raiders3:2210,raiders3:2211,raiders3:2212,raiders8:2213,raiders8:2214,raiders8:2215,raiders8:2216' --initial_learning_rate=$3 --momentum=$4 --sync=True --compute_groups=$5
